src/bin/casr-cluster.rs

```diff
@@ -450,7 +514,24 @@ fn main() -> error::Result<()> {
                 .min_values(1)
                 .max_values(2)
                 .value_name("INPUT_DIR> <OUTPUT_DIR")
-                .help("Deduplicate CASR reports. If two directories are set, deduplicated reports are copied to the second directory. If one directory is provided, duplicated reports are deleted."),
+                .help(
+                    "Deduplicate CASR reports. If two directories are set, \
+                    deduplicated reports are copied to the second directory. \
+                    If one directory is provided, duplicated reports are deleted.",
+                ),
+        )
+        .arg(
+            Arg::new("merge")
+                .short('m')
+                .long("merge")
+                .takes_value(true)
+                .min_values(2)
+                .max_values(2)
+                .value_names(&["SUB_DIR", "MAIN_DIR"]) // Maybe rename SUB_DIR into ?
```

Comment:
Maybe just input and output directories?

Reply:
NEW_DIR, STORAGE_DIR